native_client/README.md

```diff
@@ -1,105 +1,137 @@
+* [Mozilla's TensorFlow `r1.13` branch](https://github.com/mozilla/tensorflow/tree/r1.13)
+* [General TensorFlow requirements](https://www.tensorflow.org/install/install_sources)
+* [libsox](https://sourceforge.net/projects/sox/)
+
+It is required to use our fork of TensorFlow since it includes fixes for common problems encountered when building the native client files.
 
-We provide pre-built binaries for Linux and macOS.
+If you'd like to build the language bindings or the decoder package, you'll also need:
+
+* [SWIG](http://www.swig.org/)
+* [node-pre-gyp](https://github.com/mapbox/node-pre-gyp) (for Node.JS bindings only)
 
-## Required Dependencies
 
-Running inference might require some runtime dependencies to be already installed on your system. Those should be the same, whatever the bindings you are using:
-* libsox2
-* libstdc++6
-* libgomp1
-* libpthread
+## Dependencies
 
-Please refer to your system's documentation on how to install those dependencies.
+If you follow these instructions, you should compile your own binaries of DeepSpeech (built on TensorFlow using Bazel).
 
-## Installing our Pre-built Binaries
+For more information on configuring TensorFlow, read the docs up to the end of ["Configure the Build"](https://www.tensorflow.org/install/source#configure_the_build).
 
-To download the pre-built binaries, use `util/taskcluster.py`:
+### TensorFlow: Clone & Checkout
+
+Clone our fork of TensorFlow and checkout the correct version:
 
 ```
-python util/taskcluster.py --target /path/to/destination/folder
+git clone git@github.com:mozilla/tensorflow.git
+git checkout origin/r1.13
 ```
 
-If you need binaries which are different than current master (e.g. `v0.2.0-alpha.6`) you can use the `--branch` flag:
+### TensorFlow: Dependencies
+
+Prepare your environment so you can TensorFlow:
 
-```bash
-python3 util/taskcluster.py --branch "v0.2.0-alpha.6"
+```
+sudo apt install python3-dev python3-pip
+pip install -U --user pip six numpy wheel setuptools mock
+pip install -U --user keras_applications==1.0.6 --no-deps
+pip install -U --user keras_preprocessing==1.0.5 --no-deps
 ```
 
-`util/taskcluster.py` will download and extract `native_client.tar.xz`.  `native_client.tar.xz` includes (1) the `deepspeech` binary and (2) associated libraries. `taskcluster.py` will download binaries for the architecture of the host by default, but you can override that behavior with the `--arch` parameter. See `python util/taskcluster.py -h` for more details.
+### Bazel: Download & Install 
 
-If you want the CUDA capable version of the binaries, use `--arch gpu`. Note that for now we don't publish CUDA-capable macOS binaries.
+[Install the correct version of Bazel](https://docs.bazel.build/versions/master/install.html) for this TensorFlow release. For TensorFlow `r1.13` you will need [Bazel version `0.21.0`](https://github.com/bazelbuild/bazel/releases/tag/0.21.0) or lower.
```

Comment:
TensorFlow documents Bazel 0.19.2 for TF 1.13, which is also what we use in the automation. Again, just link to the upstream docs instead of copying info here: https://www.tensorflow.org/install/source#tested_build_configurations

Reply:
fixed: 7ab0962b3d31c0d9f89238da2962c2edce998e7b